central/src/components/Games.jsx

```diff
@@ -134,17 +131,17 @@ export default function Games({
         <Route path="/">
           <Grid item xs={12} className={classes.contentGrid}>
             <Box className={classes.actions}>
-              <SearchBar setSearchInput={setSearchInput} searchInput={searchInput} isSearchClick={isSearchClick} handleSearchClick={handleSearchClick} isResolutionMobile={isResolutionMobile} />
-              <SortByDropdown handleSortChange={handleSortChange} sortByCheck={sortByCheck} setSortByCheck={setSortByCheck} isResolutionMobile={isResolutionMobile} style={{zIndex: 5}}/>
+              <SearchBar isGames={location.pathname === "/"} handleSearchChange={handleSearchChange} searchInput={searchInput} isSearchClick={isSearchClick} handleSearchClick={handleSearchClick} isResolutionMobile={isResolutionMobile} />
+              <SortByDropdown isGames={location.pathname === "/"} listQuerySettings={listQuerySettings} handleUpdateListQuerySettings={handleUpdateListQuerySettings} sortByCheck={sortByCheck} setSortByCheck={setSortByCheck} style={{zIndex: 5}}/>
             </Box>
```

Comment:
Because we have a number of different sort and filter settings, I've integrated them all into a `listQuerySettings` function that is run whenever the settings up. We should only ever have one query settings object so we can accurately reset things like `nextTokens` without worrying about updating it in multiple places.